### ISSUE TITLE
clamping in model

### ISSUE THREAD

**Issue by talosh** [hzwer/arXiv2020-RIFE#118]
Hi Hzwer, hope you're well.
I wonder would it be useful to move 0, 1 clamping from model to the inference logic?
In vfx world it is normal to have values over 1 in images and it should be preserved when deaing with exr images
Thank you!

**Comment by hzwer** [comment]
Hi talosh! I think it's easy to add a flag to control that clamp. I want to know what's the value range in vfx world? Is it possible to normalize all the values to 0~1?

**Comment by talosh** [comment]
It is normal to have bright areas hitting 3-4 and even more in the bright areas in linear space. If I just comment out line 191 with pred = torch.clamp(pred, 0, 1) it works perfectly fine, the question is rather do you think it should happen in the model or in, say, inference_image or inference_video code when we're more aware of the nature of incoming images.
